cmd/cli/plugin/package/README.md

```diff
@@ -8,46 +8,65 @@ Package and Repository Operations ( Subject to Change ):
 
 ```sh
 >>> tanzu package --help
-Tanzu package management
+Tanzu package management (available, install INSTALLED_PACKAGE_NAME --package-name PACKAGE_NAME --version VERSION, installed, repository)
 
 Usage:
+  tanzu package [flags]
   tanzu package [command]
 
 Available Commands:
-    available   Manage available packages
-    install     Install a package
-    installed   Manage installed packages
-    repository  Manage registered package repositories
-
+  available     Available packages (get PACKAGE_NAME or PACKAGE_NAME/VERSION, list or list PACKAGE_NAME)
+  completion    Generate the autocompletion script for the specified shell
+  install       Install package
+  installed     Installed packages (create INSTALLED_PACKAGE_NAME --package-name PACKAGE_NAME --version VERSION, delete INSTALLED_PACKAGE_NAME, get INSTALLED_PACKAGE_NAME, list, update INSTALLED_PACKAGE_NAME)
```

Comment:
Suggestion: would it be improved the readability to show each subcommand e.g. for ```installed``` in a separate line instead of comma-separated list? (same comment or available & repository commands)

Reply:
I totally agree on this. But again this is auto generated just like the help header. I will try to make this more readable.